src/components/application_manager/include/application_manager/resumption/resume_ctrl_impl.h

```diff
@@ -138,6 +139,30 @@ class ResumeCtrlImpl : public ResumeCtrl,
    */
   void OnAwake() OVERRIDE;
 
+  /**
+   * @brief Returns Low Voltage signal timestamp
+   * @return Low Voltage event timestamp if event LOV VOLTAGE event occures
+   * otherwise 0
+   */
+  time_t LowVoltageTime() const OVERRIDE;
```

Comment:
@AByzhynar please avoid duplication of comments in implementation and interface

Reply:
@LuxoftAKutsan Done. Sorry, commit squashed to old one.